### PR TITLE
Modifying build script to add com components for test impact collector

### PR DESCRIPTION
**Pull request by prawalagarwal**
For test impact collector to work, the com components have to be present in Extensions/TestImpact folder. This change modifies the build script to achieve that.

### CODE REVIEW

File: scripts/build.ps1 [hunk 1]
```diff
@@ -323,6 +323,8 @@ function Create-VsixPackage
     $vsixProjectDir = Join-Path $env:TP_OUT_DIR "$TPB_Configuration\VSIX"
     $packageDir = Get-FullCLRPackageDirectory
     $testhostPackageDir = Join-Path $packageDir "TestHost"
+    $extensionsPackageDir = Join-Path $packageDir "Extensions"
+    $testImpactComComponentsDir = Join-Path $extensionsPackageDir "TestImpact"
```
Comment (Faizan2304) [inline scripts/build.ps1]: Who and how will these comcomponent will get consumed ?

**Comment by Faizan2304** [comment]
We have to update [TestPlatform.csproj](https://github.com/Microsoft/vstest/blob/master/src/package/VSIXProject/TestPlatform.csproj#L102) to include new added com component in TestPlatform.Vsix

**Review by Faizan2304 — CHANGES_REQUESTED** [review]

File: scripts/build.ps1 (hunk 1 above)
Comment (prawalagarwal) [inline scripts/build.ps1]: This is getting consumed by test impact collector. The way collector is implemented today requires us to put com components in Extensions/TestImpact directory

**Review by Faizan2304 — APPROVED** [review]
Please create vsix and do some manually testing.
1) Make sure com component dlls are getting drop properly
2) Test impact is working.

Rest look good to me 👍

**Review by harshjain2 — CHANGES_REQUESTED** [review]
Holding this PR until we have Test Impact DataCollector related changes in TraceCollector.dll

**Review by harshjain2 — APPROVED** [review]

**Pull request merged**